test/OrchardCore.Tests/Modules/OrchardCore.Email/Workflows/EmailTaskTests.cs

```diff
@@ -67,8 +67,10 @@ private static DefaultEmailService CreateSmtpService(SmtpOptions smtpOptions)
             var smtp = new SmtpEmailProvider(options.Object, emailValidator.Object, logger.Object, localizer.Object);
 
             var resolver = new Mock<IEmailProviderResolver>();
+#pragma warning disable CA2012 // Use ValueTasks correctly
```

Comment:
Document in a comment why this suppression is appropriate.

Reply:
I cleaned it up.